pysb/tests/test_simulator_kappa.py

```diff
@@ -0,0 +1,66 @@
+from pysb.simulator import KappaSimulator
+from pysb.kappa import run_simulation
+from pysb.examples import michment
+from pysb.bng import generate_equations
+import numpy as np
+
+
+KAPPA_SEED = 123
+
+
+def _compare_kappa_sims(run_simulation_output, kappasimulator_output):
+    sim1 = run_simulation_output.view('<f8')[:, 1:]
+    sim2 = kappasimulator_output.observables.view('f8').reshape(sim1.shape)
+
+    assert np.allclose(sim1, sim2, atol=1e-16, rtol=1e-16)
+
+
+def test_kappa_sim_michment():
+    orig_sim = run_simulation(michment.model, time=100, points=100, seed=123)
```

Comment:
I figured we would deprecate `pysb.kappa.run_simulation` as part of this change. If we do that, it probably doesn't make sense to test against it. Thoughts?

Reply:
I've now added perturbation support to `KappaSimulator` and deprecated `pysb.kappa.run_simulation`. I don't think there's any need to add flux map support, since users can use the dedicated function for that.

I'd suggest leaving the unit tests in place for deprecated code until it's actually removed.

Reply:
I see; apologies for the misunderstanding. I'll work on a new PR to remove the tests' dependence on deprecated code before the PySB 2 update.